content/en/real_user_monitoring/guide/using-session-replay-as-a-key-tool-in-post-mortems.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Using Session Replay As A Key Tool In Post-Mortems
+kind: guide
+further_reading:
+- link: 'https://www.datadoghq.com/blog/analyze-user-experience-frustration-signals-with-rum/'
+  tag: 'blog'
+  text: 'Detect user pain points with Datadog Frustration Signals'
```

Comment:
Doublecheck the spacing, but suggestion to add Notebooks docs to further reading!
```suggestion
  text: 'Detect user pain points with Datadog Frustration Signals'
- link: 'https://docs.datadoghq.com/notebooks/'
  tag: 'documentation'
  text: 'Notebooks'
```